lib/katello/permissions/capsule_content_permissions.rb

```diff
@@ -7,4 +7,10 @@
                                                     :sync, :sync_status, :cancel_sync]
              },
              :resource_type => 'SmartProxy'
+
+  permission :view_smart_proxies,
```

Comment:
Does `view_smart_proxies` not already exist in Foreman? If so, this will overwrite it, instead this would want to do how hosts permissions are handled -- https://github.com/Katello/katello/pull/5776/files#diff-42113007876025eca75222078efbb9f3R3